src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -139,7 +139,17 @@ def _set_artifact_paths(args: argparse.Namespace) -> argparse.Namespace:
     Set paths for all the artifacts.
     """
     if args.artifact_dir == Path(DEFAULT_ARTIFACT_DIR):
-        name = [f"{args.model}"]
+        # Preprocess Huggingface model names that include '/' in their model name.
+        if (args.model is not None) and ("/" in args.model):
+            filtered_name = args.model.split("/")[-1]
```

Comment:
Should we instead replace the `/` with something (e.g. `_`) so that the specificity isn't lost?

Reply:
Ooo yeah I think that's a better approach.